.gitlab-ci.yml

```diff
@@ -50,6 +50,8 @@ test:modules-artifact-gen:
     - echo "deb [arch=$(dpkg --print-architecture)] https://downloads.mender.io/repos/debian ubuntu/jammy/experimental main" | tee /etc/apt/sources.list.d/mender.list
     - apt update && apt install -yy mender-artifact
     # Test dependencies
+    - pip uninstall pytest || true
+    - pip install pytest
```

Comment:
Why?

Reply:
`TypeError: required field "lineno" missing from alias` totally out of the blue